bindings/python/IK/src/JointTrackingTask.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file JointTrackingTask.cpp
+ * @authors Paolo Maria Viceconte
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include <BipedalLocomotion/System/LinearTask.h>
+#include <BipedalLocomotion/IK/JointTrackingTask.h>
+#include <BipedalLocomotion/bindings/IK/JointTrackingTask.h>
+
+namespace BipedalLocomotion
+{
+namespace bindings
+{
+namespace IK
+{
+
+void CreateJointTrackingTask(pybind11::module& module)
+{
+    namespace py = ::pybind11;
+    using namespace BipedalLocomotion::IK;
+    using namespace BipedalLocomotion::System;
+
+    py::class_<JointTrackingTask, std::shared_ptr<JointTrackingTask>, LinearTask>(module, "JointTrackingTask")
+        .def(py::init())
+        .def("initialize",
+            [](JointTrackingTask& impl,
+               std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler>
+                   paramHandler) -> bool { return impl.initialize(paramHandler); },
+            py::arg("paramHandler"))
```

Comment:
Could you please keep the snake case?

Reply:
Sorry for the camel-case leftovers, addressed via https://github.com/dic-iit/bipedal-locomotion-framework/pull/303/commits/4076530e75aa90653d573da14637a9693d1c4aa3